cg/services/order_validation_service/workflows/microsalt/validation/inter_field/utils.py

```diff
@@ -0,0 +1,35 @@
+from cg.models.orders.sample_base import ContainerEnum
+from cg.services.order_validation_service.errors.sample_errors import OccupiedWellError
+from cg.services.order_validation_service.workflows.microsalt.models.order import MicrosaltOrder
+from cg.services.order_validation_service.workflows.microsalt.models.sample import MicrosaltSample
+
+
+class PlateSamples:
```

Comment:
Nice! We might want to think about refactoring to something similar for the case_sample flow.